src/MaxMind/Db/Reader/Decoder.php

```diff
@@ -96,37 +100,37 @@ public function decode($offset)
     private function decodeByType($type, $offset, $size)
     {
         switch ($type) {
-            case 'map':
+            case self::_MAP:
                 return $this->decodeMap($size, $offset);
-            case 'array':
+            case self::_ARRAY:
                 return $this->decodeArray($size, $offset);
-            case 'boolean':
+            case self::_BOOLEAN:
                 return [$this->decodeBoolean($size), $offset];
         }
 
         $newOffset = $offset + $size;
         $bytes = Util::read($this->fileStream, $offset, $size);
         switch ($type) {
-            case 'utf8_string':
-                return [$this->decodeString($bytes), $newOffset];
-            case 'double':
+            case self::_BYTES:
+            case self::_UTF8_STRING:
+                return [$bytes, $newOffset];
+            case self::_DOUBLE:
                 $this->verifySize(8, $size);
 
                 return [$this->decodeDouble($bytes), $newOffset];
-            case 'float':
+            case self::_FLOAT:
                 $this->verifySize(4, $size);
 
                 return [$this->decodeFloat($bytes), $newOffset];
-            case 'bytes':
-                return [$bytes, $newOffset];
-            case 'uint16':
-            case 'uint32':
-                return [$this->decodeUint($bytes), $newOffset];
-            case 'int32':
-                return [$this->decodeInt32($bytes), $newOffset];
-            case 'uint64':
-            case 'uint128':
-                return [$this->decodeBigUint($bytes, $size), $newOffset];
+
+                return [$this->decodeUint($bytes, $size), $newOffset];
```

Comment:
I think we'd want to delete this line as well!

Reply:
Woops! Thanks!